Client/Frontend/Strings.swift

```diff
@@ -150,3 +150,19 @@ extension Strings {
     public static let ThirdPartySearchFailedTitle = NSLocalizedString("Search.ThirdPartyEngines.FailedTitle", value: "Failed", comment: "A title explaining that we failed to add a search engine")
     public static let ThirdPartySearchFailedMessage = NSLocalizedString("Search.ThirdPartyEngines.FailedMessage", value: "The search provider could not be added.", comment: "A title explaining that we failed to add a search engine")
 }
+
+// Bookmark Management
+extension Strings {
+    public static let BookmarksTitle = NSLocalizedString("Bookmarks.Title", value: "Title", comment: "The label for the title of a bookmark")
+    public static let BookmarksURL = NSLocalizedString("Bookmarks.URL", value: "URL", comment: "The label for the URL of a bookmark")
+    public static let BookmarksFolder = NSLocalizedString("Bookmarks.Folder", value: "Folder", comment: "The label to show the location of the folder where the bookmark is located")
+    public static let BookmarksNewFolder = NSLocalizedString("Bookmarks.NewFolder", value: "New Folder", comment: "The button to create a new folder")
+    public static let BookmarksFolderName = NSLocalizedString("Bookmarks.FolderName", value: "Folder Name", comment: "The label for the title of the new folder")
+    public static let BookmarksFolderLocation = NSLocalizedString("Bookmarks.FolderLocation", value: "Location", comment: "The label for the location of the new folder")
```

Comment:
Just as a convention we've been adding how this string will be used in the value parameter. For example, since these are all labels I would change `Bookmarks.FolderLocation` to `Bookmarks.FolderLocation.Label`.